requirements-dev.txt

```diff
@@ -1,2 +1,34 @@
+arrow==0.12.1
+backcall==0.1.0
+beautifulsoup4==4.6.0
+cachetools==2.1.0
+certifi==2018.10.15
+chardet==3.0.4
+decorator==4.3.0
+google-api-python-client==1.7.4
+google-auth==1.5.1
+google-auth-httplib2==0.0.3
+httplib2==0.11.3
+idna==2.7
 ipdb==0.11
--r requirements.txt
+ipython==7.1.1
+ipython-genutils==0.2.0
+jedi==0.13.1
+oauth2client==4.1.2
+parso==0.3.1
+pexpect==4.6.0
+pickleshare==0.7.5
+prompt-toolkit==2.0.7
+ptyprocess==0.6.0
+pyasn1==0.4.4
+pyasn1-modules==0.2.2
+Pygments==2.2.0
+python-dateutil==2.7.5
+requests==2.19.1
+rsa==4.0
+selenium==3.141.0
+six==1.11.0
+traitlets==4.3.2
+uritemplate==3.0.0
+urllib3==1.23
+wcwidth==0.1.7
```

Comment:
looks like you put production requirements in the dev file, can we have that fixed please?

Reply:
Ok, i'll work on it, got confused